ons_alpha/bulletins/models.py

```diff
@@ -164,3 +163,14 @@ def previous_releases(self, request):
             context_overrides={"bulletins": previous},
             template="templates/pages/bulletins/previous_releases.html",
         )
+
+    @path("previous/v<int:version>/")
+    def previous_version(self, request, version):
+        page_revision = get_object_or_404(Revision, pk=version)
```

Comment:
@RealOrangeOne  - Thank you for the feedback. I've now added the necessary validation to ensure page_revision is a revision for the current page using Django's content types. This will validate the revision in the database, ensuring it is of the correct page type and is published.

Reply:
I'm not seeing any code which does this validation? 

With that said, it's probably possible to avoid that need entirely if the `version` is not the primary key, as the rest of the validation is already done when populating the field.